Tevling/Pages/Dashboard.razor.cs

```diff
@@ -0,0 +1,118 @@
+namespace Tevling.Pages;
+
+public partial class Dashboard : ComponentBase {
+
+    [Inject]
+    IAuthenticationService AuthenticationService { get; set; } = null!;
+
+    [Inject]
+    IAthleteService AthleteService { get; set; } = null!;
+
+    [Inject]
+    IChallengeService ChallengeService { get; set; } = null!;
+
+
+    private Athlete? Athlete { get; set; }
+    private Dictionary<Challenge, (string, string)> ActiveChallenges { get; set; }  = [];
+    private Dictionary<Challenge, (string, string)> RecentOutdatedChallenges { get; set; } = [];
+    private IEnumerable<Athlete> SuggestedAthletes {get; set; } = [];
+    
+    protected override async Task OnInitializedAsync()
+    {
+        Athlete = await AuthenticationService.GetCurrentAthleteAsync();
+
+        if (Athlete != null)
+        {
+            int athleteId = Athlete.Id;
+            await FetchActiveChallenges();
+            await FetchRecentOutdatedChallenges();
+            await FetchSuggestedAthletes();
+        }
+    }
+    
+    private string GetGreeting() {
+        if (Athlete is null) return string.Empty;
+
+        string athleteFirstName = Athlete.Name.Split(' ')[0];
+        return DateTime.Now.Hour switch {
```

Comment:
You should use `BrowserTime` instead of `DateTime`. Also other occurrences in this file.